handsontable.d.ts

```diff
@@ -1974,7 +1974,7 @@ declare namespace Handsontable {
 
   namespace RecordTranslation {
     interface IndexMap {
-      getValues: number[],
+      getValues: () => any[],
```

Comment:
```suggestion
      getValues: () => any[] | any[],
```

Reply:
Does it seem that the method returns not only functions with different return signatures but returns different primitives as well as?